From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

doctests have a minor failure when using ghc-9.0.2. Disabling for now.

    Test suite doctests: RUNNING...
    /var/tmp/portage/dev-haskell/higgledy-0.4.1.1-r3/work/higgledy-0.4.1.1/src/Data/Generic/HKD.hs:91: failure in expression `deconstruct ("Hello", True) ^. position @4'
    expected: ...
              ... error:
              ... The type HKD
              ... ([Char], Bool) f does not contain a field at position 4
              ...
     but got:
              <interactive>:53:1: error:
                  • The type HKD
                               (String, Bool) f does not contain a field at position 4
                                ^
                  • When checking the inferred type
                      it :: forall {f :: * -> *}.
                            ((TypeError ...), Applicative f) =>
                            f Bool
    
    Examples: 42  Tried: 42  Errors: 0  Failures: 1
    Test suite doctests: FAIL

diff -urN higgledy-0.4.1.1/higgledy.cabal higgledy-0.4.1.1-r1/higgledy.cabal
--- higgledy-0.4.1.1/higgledy.cabal	2022-02-10 23:14:54.255163552 -0700
+++ higgledy-0.4.1.1-r1/higgledy.cabal	2022-02-10 23:15:32.785092598 -0700
@@ -36,6 +36,7 @@
   default-language:    Haskell2010
 
 test-suite doctests
+  Buildable: False
   build-depends: base
                , base-compat ^>= 0.11
                , doctest ^>= 0.17
